x-pack/plugins/triggers_actions_ui/public/application/sections/alerts_table/alerts_table_state.tsx

```diff
@@ -318,13 +318,18 @@ const AlertsTableStateWithQueryProvider = ({
   const CasesContext = casesService?.ui.getCasesContext();
   const isCasesContextAvailable = casesService && CasesContext;
 
+  const memoizedCases = useMemo(
+    () => ({
+      data: cases ?? new Map(),
+      isLoading: isLoadingCases,
+    }),
+    [cases, isLoadingCases]
+  );
+
   const tableProps: AlertsTableProps = useMemo(
     () => ({
       alertsTableConfiguration,
-      cases: {
-        data: cases ?? new Map(),
-        isLoading: isLoadingCases,
-      },
```

Comment:
It looks like even if tableProps is memoized, `tableProps.cases` is calculated everytime and it results in `tableProps.cases` being updated and re-rendering all children.

Reply:
could we test this somehow?